<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>321</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>251</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="globalLayout_p1" stretch="0,0,0,0">
        <item>
         <widget class="QLabel" name="label_welcome">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Welcome</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_select_data">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select data type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_box_data_type">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>int</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>string</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>char</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Select tree type:</string>
              </property>
              <widget class="QWidget" name="verticalLayoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>40</y>
                 <width>201</width>
                 <height>140</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QRadioButton" name="radio_button_splaytree">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Splay Tree</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radio_btn_btree">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>B Tree</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radio_button_bplustree">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>B+ Tree</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>771</width>
        <height>627</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>400</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>screen</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: solid 2px;</string>
           </property>
           <property name="text">
            <string>ForwardIteration ReverseIteration</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>400</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWelcome_Page">
    <property name="title">
     <string>tools</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_to_start_page"/>
    <addaction name="actionreset_window"/>
   </widget>
   <addaction name="menuWelcome_Page"/>
  </widget>
  <action name="action_to_start_page">
   <property name="text">
    <string>&lt;- to start page</string>
   </property>
  </action>
  <action name="actionreset_window">
   <property name="text">
    <string>reset window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
